doc/how-to-contribute.rst

```diff
@@ -56,3 +56,94 @@ favourite tool easily.
 .. _Sphinx: http://www.sphinx-doc.org/en/stable
 .. _PyLint: https://www.pylint.org
 .. _Codacy: https://www.codacy.com/app/tango-controls/pytango/dashboard
+
+Releasing a new version
+-----------------------
+
+From time to time a new version should be released.  The basic steps required are
+as follows:
+
+Pick a version number
+  * Semantic version numbering is used:  <major>.<minor>.<patch>
+  * The major and minor version fields (9.2) track the TANGO C++ core version.
+  * For small changes, increase the patch field (last number), for example:
+    9.2.4 -> 9.2.5
```

Comment:
For patch releases, the version number should correspond the current development number since each release process finishes with a version bump, e.g:
-  9.2.5.devN or 9.2.5aN or 9.2.5bN (current development branch)
- to 9.2.5 (the actual release)
- to 9.2.6.dev0 (bump the patch version at the end of the release process)